activerecord/test/cases/tasks/mysql_rake_test.rb

```diff
@@ -96,15 +96,15 @@ def teardown
       end
 
       def test_raises_error
-        assert_raises(Mysql2::Error) do
+        assert_raises(Mysql2::Error, "Invalid permissions") do
           ActiveRecord::Tasks::DatabaseTasks.create @configuration
         end
       end
     end
 
     class MySQLDBDropTest < ActiveRecord::TestCase
       def setup
-        @connection    = stub(drop_database: true)
+        @connection    = Class.new { def drop_database(name); true end }.new
```

Comment:
looks like returing `true` doesn't make any effect on test cases, so i think we can omit returing `true` too.
```diff
diff --git a/activerecord/test/cases/tasks/mysql_rake_test.rb b/activerecord/test/cases/tasks/mysql_rake_test.rb
index ff2d66b17e..155b060b1e 100644
--- a/activerecord/test/cases/tasks/mysql_rake_test.rb
+++ b/activerecord/test/cases/tasks/mysql_rake_test.rb
@@ -104,7 +104,7 @@ def test_raises_error

     class MySQLDBDropTest < ActiveRecord::TestCase
       def setup
-        @connection    = Class.new { def drop_database(name); true end }.new
+        @connection    = Class.new { def drop_database(name); end }.new
         @configuration = {
           "adapter"  => "mysql2",
           "database" => "my-app-db"
```